spi.java.hints/apichanges.xml

```diff
@@ -25,6 +25,20 @@
         <apidef name="JavaHintsSPI">Java Hints SPI</apidef>
     </apidefs>
     <changes>
+        <change id="JavaFixUtilities.isPrimary">
+            <api name="JavaHintsSPI"/>
+            <summary>Added JavaFixUtilities.isPrimary() utility</summary>
+            <version major="1" minor="29"/>
```

Comment:
Nice to see the API change here! Shouldn't you guys also increase version in `manifest.mf` and reference it here? E.g. use `1.30`?

Reply:
Hello @jtulach. Thank you for your review.

I was a little confused by this; I figured that I needed to bump some sort of version number, but I found that the "spec.version.base" in `spi.java.hints/nbproject/project.properties` was already set to "1.30.0" in the project properties:

https://github.com/apache/incubator-netbeans/blob/dd5eb868ec5e8037c77118dbd5e2c53103f1c30e/spi.java.hints/nbproject/project.properties#L20

By the way, I have opened PR #320 to track this API change separately from the "Add a new fix to BoxedIdentityComparison" changes.